autoload/vimteractive.vim

```diff
@@ -103,9 +107,13 @@ function! vimteractive#sendlines(lines)
     endif
 endfunction
 
+function! vimteractive#get_output()
```

Comment:
I thinks this function is an accident? It’s empty and it looks like it’s not referenced anywhere.

Reply:
Indeed -- Removed in a32bff4